Community/Tdarr_Plugin_vdka_Tiered_CPU_CRF_Based_Configurable.js

```diff
@@ -222,8 +422,10 @@ const plugin = (file, librarySettings, inputs, otherArguments) => {
 
   // encoding settings
   response.preset += `,-map 0 -dn -c:v libx265 -preset ${ffmpegPreset}`
-  + ` -x265-params crf=${crf}:bframes=${bframe}:rc-lookahead=32:ref=6:b-intra=1:aq-mode=3`
-  + ' -a53cc 0 -c:a copy -c:s copy -max_muxing_queue_size 9999';
+  + ` ${use10Bit} -x265-params crf=${crf}:qpmax=40:bframes=${bframe}:rc-lookahead=32:ref=6:b-intra=1:aq-mode=3`
+  + ' -a53cc 0 -c:a copy -c:s copy -metadata:g title= -metadata:s:v:0 titel= -metadata:s:v:0 name= '
```

Comment:
`-metadata:g title= -metadata:s:v:0 titel= -metadata:s:v:0 name=` there's a typo in there, plus it isn't mentioned anywhere. So either remove it or document it.

Reply:
The plugin should absolutely not touch metadata.